test/d-b-n_smoke/arrays.html

```diff
@@ -212,7 +243,7 @@
                         ppclient.addEventListener('patch-applied', patchAppliedEventCallback);
                         // change model
                         // puppetChange(puppet, [operationObject('add', '/people_properties/-', '1')]);
-                        puppetChange(puppet, [operationObject('add', '/people_properties/0', '1')]);
+                        puppetChange(puppet, [operationObject('add', '/people_properties/0', 'property')]);
```

Comment:
You are not adding a property, are you? I'd rename it to 'item'/'element'. It was really confusing to me to figureout what you are doing here.